crates/store/re_chunk_store/src/lib.rs

```diff
@@ -43,14 +43,14 @@ pub(crate) use self::store::ColumnMetadataState;
 
 // Re-exports
 #[doc(no_inline)]
-pub use re_chunk::{
-    Chunk, ChunkId, ChunkShared, LatestAtQuery, RangeQuery, RangeQueryOptions, RowId,
-    UnitChunkShared,
+pub use {
+    re_chunk::{
+        Chunk, ChunkId, ChunkShared, LatestAtQuery, RangeQuery, RangeQueryOptions, RowId, Span,
+        UnitChunkShared,
+    },
+    re_log_types::{ResolvedTimeRange, TimeInt, TimeType, Timeline},
```

Comment:
Same comment as above re re-export, though I see we have an established pattern of doing that 😓 So this might be a taste issue....